baseband/vdif/header.py

```diff
@@ -212,6 +248,11 @@ def from_mark5b_header(cls, mark5b_header, bps, nchan, **kwargs):
 
     # properties common to all VDIF headers.
     @property
+    def edv(self):
+        """VDIF Extended Data Version (EDV)"""
```

Comment:
Full stop at end of docstring (this matters for sphinx, although perhaps only if there is more text).